recipes/scons/all/conanfile.py

```diff
@@ -34,7 +34,7 @@ def source(self):
     def build(self):
         with tools.chdir(self._source_subfolder):
             output = io.StringIO()
-            self.run("{} setup.py --requires".format(sys.executable), output=output)
+            self.run("python setup.py --requires", output=output)
```

Comment:
there is few more places that do this 
(cant comment there from view i have open, but line 42 and 53 )

Reply:
Thanks for checking and running the recipe.
Is this also used on your *ancient* build system?

Reply:
haha i actually lol'd.
i am still battling that subversiong thing, yes.
and thanks to your other comment i noticed my subversion build does not handle http -_-.
and serf library which is used by subversion (presumably, didnt get to use it yet ) ofc uses something called scons I just heard legends about!!

Btw, If i was doing what i am trying to do ~3months ago i'll be doomed, i should send you flowers.